src/app/record-ide/components/record-ide.component.ts

```diff
@@ -171,7 +171,7 @@ export class RecordIDEComponent implements OnInit {
   }
 
   navigateToLibraries() {
-    window.open("https://github.com/code-kern-ai/record-ide/blob/dev/requirements.txt", "_blank");
+    window.open("https://github.com/code-kern-ai/refinery-record-ide-env/blob/dev/requirements.txt", "_blank");
```

Comment:
should we maybe also make these <a> tags? this would enable us to navigate via the middle mouse button and people see where they are redirected to in the bottom left corner.

Reply:
Yes sure, it will be better